Dockerfile

```diff
@@ -17,18 +9,27 @@ RUN cargo build --workspace --release \
 
 FROM debian:stretch-slim
 ARG PROJ_VERSION
-ENV PROJ_DEB "proj_${PROJ_VERSION}_amd64.deb"
-ENV GPG_KEY "C60D758F807A525534C5DFD57B639E3638A8112A"
-VOLUME /app/input
-VOLUME /app/output
-RUN BUILD_DEPENDENCIES="apt-transport-https gnupg2 wget" \
-	&& apt update \
-	&& apt install --yes ${BUILD_DEPENDENCIES} \
-	&& wget --quiet --output-document - "https://kisiodigital.jfrog.io/kisiodigital/api/gpg/key/public" | apt-key add - \
-	&& echo "deb [arch=amd64] https://kisiodigital.jfrog.io/kisiodigital/debian-local stretch main" > /etc/apt/sources.list.d/kisio-digital.list \
-	&& apt update \
-	&& apt install --yes libtiff-dev libcurl4-nss-dev proj=${PROJ_VERSION} \
-	&& apt purge --yes ${BUILD_DEPENDENCIES} \
-	&& apt autoremove --yes \
-	&& rm -rf /var/lib/apt/lists/*
+## Inspired by Docker image `kisiodigital/rust-ci:latest-proj`
+## See https://github.com/CanalTP/ci-images/blob/master/rust/proj/Dockerfile
+# For building `libproj' and the Rust's crate `proj-sys`, the following Debian packages are needed:
+ARG BUILD_DEPENDENCIES="libcurl4-gnutls-dev libsqlite3-dev libtiff5-dev cmake pkg-config sqlite3 wget"
+# For running `proj`, the following Debian packages are needed:
+# - 'clang' provides 'llvm-config', 'libclang.so' and 'stddef.h' needed for compiling 'proj-sys'
+# - 'libtiff5' provides 'libtiff.so', needed for linking when 'proj-sys' is used
+# - 'libcurl3-nss' provides 'libcurl-nss.so', needed for linking when 'proj-sys' is used
+# - 'proj' provides 'proj.h' and 'libproj.so', needed for compiling 'proj-sys' (installed manually below)
+ARG RUNTIME_DEPENDENCIES="clang libtiff5 libcurl3-nss"
```

Comment:
Those changed, is it after work on rust-ci that you detected we could use more precise packages?
Also it looks like we got them on both debian 9 and 10, but should we test that a bit?

Reply:
I've done this work a while ago. What kind of failure do we expect that would need additional testing?

And indeed, they're available on `buster`:
- https://packages.debian.org/stable/clang
- https://packages.debian.org/stable/libcurl3-nss
- https://packages.debian.org/stable/libtiff5

Reply:
I think we should just test that all concerned binaries are running fine : are they all running on dev?

Reply:
I've made some improvement on how the `Dockerfile` build the project (see 367d943380d1fcae67d847efebbb1259f4dcef83). I was able to test that the binaries are running fine inside the container.